src/js/components/__tests__/__snapshots__/components-test.js.snap

```diff
@@ -1947,8 +1946,7 @@ boolean",
       },
       Object {
         "defaultValue": "none",
-        "description": "Spacing around the outer edge of
-    the drawing coordinate area for the graphic elements to overflow into.",
+        "description": "Item padding.",
```

Comment:
This change is interesting. Although your change was related to List component, somehow it impacted the snapshots of Chart. Can you please have a look at how we can maintain the Chart's detailed description and not override it as this PR suggests?

Reply:
Yes, fixed this in latest push